scripts/xss_linter.py

```diff
@@ -2003,7 +2003,7 @@ def _is_django_template(self, mako_template):
             True if this is really a Django template, and False otherwise.
 
         """
-        if re.search('({%.*%})|({{.*}})', mako_template) is not None:
+        if re.search('({%.*%})|({{.*}})|({#.*#})', mako_template) is not None:
```

Comment:
@cpennington: Thanks for making changes to the linter.  A couple of thoughts:
1. I don't think the linter handles django-templates appropriately, so the commit comment is misleading.  Maybe try:
```
Improve ability to discriminate between django and mako template.
Django-format comments are now used to detect a django template.
Note: Django templates are still not linted.
```
2. Everything should have a test, so I would imagine that a test should be enhanced with this change.